developer/workflow/importing-and-invoking-a-windows-powershell-workflow.md

```diff
@@ -1,137 +1,140 @@
+
+            PowerShell powershell = PowerShell.Create();
+            powershell.Runspace = runspace;
+            Console.Write("Setting $VerbosePreference=\"Continue\"...");
+            powershell.AddScript("$VerbosePreference=\"Continue\"");
+            powershell.Invoke();
+            Console.WriteLine("done");
+
+            Console.Write("Importing Workflow module...");
+            powershell.Commands.Clear();
+
+            //Import the module in to the PowerShell runspace. A XAML file could also be imported directly by using Import-Module.
+            powershell.AddCommand("Import-Module").AddArgument(moduleFile);
+            powershell.Invoke();
+            Console.WriteLine("done");
+
+            Console.Write("Creating job proxy...");
+            powershell.Commands.Clear();
+            powershell.AddCommand("Get-Proc").AddArgument("*");
+            PSJobProxy job = powershell.AsJobProxy();
+            Console.WriteLine("done");
+
+                Console.WriteLine();
+                Console.WriteLine("Using job proxy and performing operations...");
+                Console.WriteLine("State of Job :" + job.JobStateInfo.State.ToString());
+                Console.WriteLine("Starting job...");
+                job.StartJob();
+                Console.WriteLine("State of Job :" + job.JobStateInfo.State.ToString());
+
+                // use blocking enumerator to wait for objects until job finishes
+                job.Output.BlockingEnumerator = true;
+                foreach (PSObject o in job.Output)
+                {
+                    Console.WriteLine(o.Properties["ProcessName"].Value.ToString());
+                }
+
+                // wait for a random time before attempting to stop job
+                Random random = new Random();
+                int time = random.Next(1, 10);
+                Console.Write("Sleeping for {0} seconds when job is running on another thread...", time);
+                System.Threading.Thread.Sleep(time * 1000);
+                Console.WriteLine("done");
+                Console.WriteLine("Stopping job...");
+                job.StopJob();
+                Console.WriteLine("State of Job :" + job.JobStateInfo.State.ToString());
+                Console.WriteLine();
+                job.Finished.WaitOne();
+                Console.WriteLine("Output from job");
+                Console.WriteLine("---------------");
+
+                foreach (PSObject o in job.Output)
+                {
+                    Console.WriteLine(o.Properties["ProcessName"].Value.ToString());
+                }
+
+                Console.WriteLine();
+                Console.WriteLine("Verbose messages from job");
+                Console.WriteLine("-------------------------");
+                foreach (VerboseRecord v in job.Verbose)
+                {
+                    Console.WriteLine(v.Message);
+                }
+
+            runspace.Close();
+        }
+    }
+}
+
+```


```

Comment:
control character

Reply:
This was an issue with my code fence logic and is fixed